pkg/controller/clustercontroller/backups.go

```diff
@@ -18,17 +18,16 @@ package clustercontroller
 
 import (
 	"fmt"
-	"reflect"
-	"sync"
-	"time"
-
 	"github.com/golang/glog"
+	api "github.com/presslabs/mysql-operator/pkg/apis/mysql/v1alpha1"
+	myclientset "github.com/presslabs/mysql-operator/pkg/generated/clientset/versioned"
 	"github.com/wgliang/cron"
+	k8errors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
-
-	api "github.com/presslabs/mysql-operator/pkg/apis/mysql/v1alpha1"
-	myclientset "github.com/presslabs/mysql-operator/pkg/generated/clientset/versioned"
+	"reflect"
```

Comment:
Standard library imports should be the first, then third party imports and then local imports separated by a new line.